package.json

```diff
@@ -15,7 +15,20 @@
     "biojs"
   ],
   "main": "build/pileup.js",
-  "browser": "build/pileup.js",
+  "browser": "src/pileup.js",
+  "browserify": {
```

Comment:
OK -- I was able to make the shared the settings work. I am with you in terms of preferring `grunt` for the browserify task, I actually didn't even know about the use of browserify in a different setting before I had this issue.